stable/pipelines/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: pipelines
 home: https://jfrog.com/pipelines/
-version: 1.3.9
-appVersion: 1.6.2
+version: 1.4.0
+appVersion: 1.7.1
```

Comment:
Can you add this version update  in changelog ?

Reply:
sure